runners/core-java/src/test/java/org/apache/beam/runners/core/metrics/MonitoringInfoMatchers.java

```diff
@@ -76,17 +63,20 @@ public void describeTo(Description description) {
    * Matches a {@link MonitoringInfo} with that has the set fields in the provide MonitoringInfo.
    *
    * <p>This is useful for tests which do not want to match the specific value (execution times).
-   * Currently this will only check for URNs, labels, type URNs and int64Values.
+   * Currently this will only check for URNs, labels, type URNs and {@code beam:coder:varint:v1}
+   * encoded values.
    */
-  public static TypeSafeMatcher<MonitoringInfo> valueGreaterThan(final long value) {
+  public static TypeSafeMatcher<MonitoringInfo> valueGreaterThanOrEqualTo(final long value) {
```

Comment:
would this only work with certain types? Should this be renamed?

Reply:
Renamed to `counterValueGreaterThanOrEqualTo`

Also fixed comment.